### PR TITLE
Feature/label schema

### PR DESCRIPTION
**Pull request by D-Fink**
# Description
removed status from product schema
add productStatus, productHealth, state to Product type
add projectStatus, projectHealth, state to Project type
add enum Color - subject to change
add Label schema
add createLabel resolver
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, but not tested (may need new tests)

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: sourceme.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+export $(grep -v '^#' .env | xargs -0)
```
Comment (AaronShawnSoler) [inline sourceme.sh]: Put this file in your git ignore so that it doesn't get pushed to the repository.

File: apollo/src/schema.js [hunk 2]
```diff
@@ -100,6 +108,15 @@ const typeDefs = gql`
     PM
   }
 
+  enum Color {
```
Comment (AaronShawnSoler) [inline apollo/src/schema.js]: make sure to update types in apollo as well

File: apollo/src/schema.js [hunk 3]
```diff
@@ -112,6 +129,12 @@ const typeDefs = gql`
     rating: Int!
   }
 
+  type Label {
```
Comment (AaronShawnSoler) [inline apollo/src/schema.js]: added Label type but didn't add CCRepo type in apollo

Comment (D-Fink) [reply apollo/src/schema.js]: CCRepo not our bag

Comment (AaronShawnSoler) [reply apollo/src/schema.js]: okay just make sure apollo has types that it can also understand otherwise it will cause issues

**Comment by Memitaru** [comment]
Since this is a change to the schema the README needs to be edited as well to document the changes.

File: apollo/src/schema.js (hunk 3 above)
Comment (lawsonarichard) [reply apollo/src/schema.js]: > CCRepo not our bag

Curious to why CCRepo is there then. You'll need to remove it if it isn't associated with your feature.

Comment (D-Fink) [reply apollo/src/schema.js]: not sure. I did not add it but i will take it out with, hopefully, the final push

**Pull request merged**